.github/workflows/linting-icons.yml

```diff
@@ -1,58 +1,55 @@
-name: Add Changed Icons comment
+name: Linting Icons
 
 on:
-  pull_request_target:
+  pull_request:
```

Comment:
This workflow will then not run anymore on PR that are from forks :/

Reply:
Hmm then we should create dispatchable worklows to increase security.

Reply:
Okay, I will explain a bit.
We need to phase out the use of `pull_request_target ` since it is a security threat to our repo.
The use of `pull_request_target` enables running malicious scripts with repo write permissions.
See our latest security advisories.

Related blog posts about this:
- https://nathandavison.com/blog/github-actions-and-the-threat-of-malicious-pull-requests
- https://securitylab.github.com/resources/github-actions-preventing-pwn-requests/

Running this actions in forks is still possible but you need to enable github actions in your repo yourself.